kuma/core/helpers.py

```diff
@@ -150,8 +151,11 @@ def entity_decode(str):
 
 
 @register.function
-def inlinei18n(locale):
-    return StaticI18nJob().get(locale)
+def statici18n(locale):
+    if not locale:
```

Comment:
Please just use the template tag function from statici18n here (`statici18n.templatetags.statici18n.statici18n` no kidding) to reduce the code duplication. `request.locale` is guaranteed to be set so no need to fall back to `settings.LANGUAGE_CODE` (that's already done in the locale middleware) either.